src/app/components/BibPage/BibDetails.jsx

```diff
@@ -77,20 +84,19 @@ class BibDetails extends React.Component {
         {
           bibValues.map((value, i) => {
             const url = `filters[${fieldValue}]=${value['@id']}`;
-            return (
-              <li key={i}>
-                {
-                  fieldLinkable ?
-                    <Link
-                      onClick={e => this.newSearch(e, url)}
-                      to={`${appConfig.baseUrl}/search?${url}`}
-                    >
-                      {value.prefLabel}
-                    </Link>
-                    : <span>{value.prefLabel}</span>
-                }
-              </li>
-            );
+            let itemValue = fieldLinkable ?
```

Comment:
Either it's an app link or it's just regular text. The second case would be if it's an external link, or 'self link' since its prefLabel is used as the label. So yes, three cases but only one value.